### ISSUE TITLE
[1.2.0] Fails on upgrade

### ISSUE THREAD

**Issue by jaredsburrows** [fabasoad/setup-cobol-action#122]
## Description:
Fails on upgrade

## Repro / Example:
https://github.com/jaredsburrows/hello-world/pull/16

## Error:
```
Run fabasoad/setup-cobol-action@v1.2.0
Run COBOL_INSTALLED=$(if command -v cobc >/dev/null 2>&1; then echo true; else echo false; fi)
Run src/install.sh "3.1.2"
/home/runner/work/_temp/661ee1fb-2b8a-4ffc-b9c7-8ce8ffc01fbe.sh: 1: src/install.sh: not found
Error: Process completed with exit code 127.
```

**Comment by fabasoad** [comment]
@jaredsburrows Thanks for opening this issue! I've just released a new version - `v1.3.0`. It should be fixed there. Please validate.

**Comment by jaredsburrows** [comment]
@fabasoad 1.4.0 fails as well:

PR: https://github.com/jaredsburrows/hello-world/pull/17

Build: https://github.com/jaredsburrows/hello-world/actions/runs/3545003025/jobs/5952776571

**Comment by fabasoad** [comment]
@jaredsburrows It's a different problem but also a regression. Sorry for the troubles. Should be fixed in [v1.4.1](https://github.com/fabasoad/setup-cobol-action/releases/tag/v1.4.1). Please validate. Thanks.

**Comment by jaredsburrows** [comment]
@fabasoad Thanks - https://github.com/jaredsburrows/hello-world/pull/19.
